consensus/src/dag/dag_store.rs

```diff
@@ -183,7 +183,7 @@ impl Dag {
                             .any(|cert| cert.metadata() == metadata)
                     })
                     .map(|node_status| node_status.as_node().author());
-                validator_verifier.check_voting_power(votes).is_ok()
+                validator_verifier.check_voting_power(votes, false).is_ok()
```

Comment:
have a TODO to make this an onchain config? we were discussing this could be used for deterministically skipping anchors?

Reply:
later :)